release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -1533,8 +1533,29 @@ All MongoDB-based samples have been replaced, deprecated, or removed.
+
+* Previously, the users were not able to see any Knative services and sources on the topology view. This was because of the triggers possessing a Knative service and In-Memory Channel as a subscriber due to a Null Pointer Exception (NPE). This issue has now been resolved by fixing the Null Pointer Exception, so that the Knative data model returns proper data and the Knative resources are properly shown on the Topology. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1907827[*BZ#1907827*])
+
+* Previously, the API server could fail to create a resource and return a 409 status code due to a conflict while updating a resource quota resource. This issue has now been fixed and the OpenShift Web Console will attempt to retry the request while receiving a 409 status code. At most three attempts will be made which is often sufficient to complete the request. In the event that a 409 continues to occur, an error will be displayed in the console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1928230[*BZ#1928230*])
+
+* Previously, no helm charts were displayed in the developer catalog because the chart repository `httpClient()` did not consider any proxy environment variable. This issue has now been fixed and the helm charts are now being displayed in the developer catalog. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1919138[*BZ#1919138*])
+
+* Technology preview badges are being displayed on the Eventing user interface although it would be going for a GA release. The Technology preview badges are now removed from Eventing user interface and the user will no longer be able to see them. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1899382[*BZ#1899382*])
+
+* Previously, the application crashed when the correlated pod data was not available for the `DeploymentConfig`. This was because the console deployment fetched two sets of data with the pod status donut shown as soon as `DeploymentConfig` is loaded. If the API returns more than 250 pods, some of the information is skipped and is not available. This issue has now been fixed and the pod data is available even when the project contains more than 250 pods, thus ensuring that the `DeploymentConfig` detail page does not crash anymore. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1921603[*BZ#1921603*])
+
+* Previously, the static models corresponding to trigger/subscription/channel/IMC were having the `apiVersion` with beta. With the serverless 0.10, the latest supported versions got updated for trigger/subscription/channel/IMC. This bug fix updates the apiVersion for eventing resources to v1 and the user interface models point to the latest supported version. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1896625[*BZ#1896625*])
```

Comment:
- Replace: 'trigger/subscription/channel/IMC were having the `apiVersion` with beta.' with: 'the trigger, subscription, channel, and IMC eventing sources used the beta api version.
- With the Serverless 0.10 release, the latest supported versions for eventing sources were updated to the v1 version.
- This bug fix updates the user interface models to point to the latest supported version.

Reply:
Done!